windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -21,13 +21,21 @@ ifndef::openshift-origin[]
 
 {productwinc} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```

Comment:
Is "Windows Container Support" different from"Red Hat OpenShift Support for Windows Containers" and "Windows Container Support for Red Hat OpenShift"? I noticed that all three phrases are used and capitalised in this assembly. You could potentially only use one of these names in the assembly for consistency or provide the reader with explicit clarification about what they both refer to. Are these all official product names? I could only find "Red Hat OpenShift for Windows Containers" in the product name spreadsheet.

My assumption from this line is that it's the support aspect that requires the additional subscription. You could clarify further if the subscription is required to receive support for Red Hat OpenShift for Windows Containers, or to both install the product and receive support from Red Hat. But, from the preceding paragraph it looks like the subscription is required to use the product.

Reply:
You could update the URL to direct readers to the main support page on the Customer Portal, https://access.redhat.com/support/.